node/errors.js

```diff
@@ -563,6 +607,7 @@ module.exports.classify = function classify(err) {
         case 'tchannel.max-pending-for-service':
         case 'tchannel.no-peer-available':
         case 'tchannel.no-service-handler':
+        case 'tchannel.request.retry-limit-exceeded':
```

Comment:
Note: If we return `Cancelled` for retry-limit-exceeded we could enable retries at exit node; this would propagate a Cancelled when retries run out and the entry node or edge clients would not retry in a cascading fashion.